gulpfile.babel.js

```diff
@@ -387,7 +388,7 @@ const exportSassVendor = () =>
     .pipe(gulp.dest(config.dir.export.sassVendor));
 
 const exportImages = () =>
-  gulp.src(`${config.dir.build.images}/**/*`)
+  gulp.src(config.files.src.imagesExportable)
```

Comment:
This change goes back to the source, rather than the built images, so the results are now no longer optimised.